Workflows/GIT/Pull requests.md

```diff
@@ -0,0 +1,64 @@
+Pull requests initiate discussion about your commits. Since they're tightly integrated with the underlying Git repository, everyone can see exactly what changes will be merged if they accept your request.
+
+## New pull request
+Once a feature or fix is done, a PR for it should be opened.
+
+It's essential to write a good pull request description. Reviewers will usually read it before
+looking at the diff, so make sure it gives them enough context to know what they are looking at.
+
+Another purpose of a good description is to provide the documentation for a future reference.
+A new developer might join the project in the future. The description will help him understand
+the motivation behind the implementation of a specific feature. Please note that any important information
+from the description should also be added to the main documentation in the project as a central reference point
+where the latest state of the project is documented.
+
+A new Rails project created with the [Infinum rails template](https://github.com/infinum/default_rails_template)
+includes a [pull request description template](https://help.github.com/en/articles/creating-a-pull-request-template-for-your-repository).
+The template is defined in the `rails_project/.github/PULL_REQUEST_TEMPLATE.md` file.
```

Comment:
We talked on slack about adding a default title template as well. 

I usually use:
`Task #{task numbers} - Short description` if PR is for one or two tasks

or

`Task #tag - Short description` - if PR is for a bigger feature where all tasks are tagged with #tag

<img width="981" alt="Pull_Requests_·_infinum_web-loop-community-backend" src="https://user-images.githubusercontent.com/6377753/87007238-38dc4f80-c1c2-11ea-9625-abf49b03348f.png">

It's easier to find a PR you're looking for like this, rather than trying to match task title with PR title.

Reply:
Could you update the PR template in default_rails_template with this?

Reply:
I usually go to the Productive task and try finding the PR link there. I've always been opening PRs like that 😄 

Going to the PR list on Github and searching sounds slow if there's a lot of them.

Reply:
@melcha I think you can only edit the PR description, and the template is for that, not the title. I might be mistaken and there is some solution to have this automatic 🤔

Reply:
I actually first go to Github and see the PRs there. Am using productive less and less these days :D